less/index.less

```diff
@@ -52,8 +52,345 @@ table {
+                            font-size: 1.4rem;
+                            border-top: none;
+                            border-bottom: 2px solid @clr-light-grey;
+                        }
+                        &:nth-child(even) {
+                            background-color: rgba(22, 22, 22, .1);
+                        }
+                        td {
+                            padding: 25px 5px;
+                            &:first-child {
+                                font-weight: bold;
+                            }
+                        }
+                    }
+                }
+
+                /*bottom-content */
+                #contact {
+                    display: flex;
+                    flex-wrap: wrap;
+                    margin: 20px 0px 15px;
+                    h3 {
+                        margin-bottom: 20px;
+                    }
+                    form {
+                        .column-left;
+                        width: 300px;
+                        input, textarea {
+                            padding: 10px;
+                            border: 1.5px solid CornflowerBlue;
+                        }
+                        input {
+                            margin-bottom: 20px;
+                        }
+                    }
+                    .btn {
+                        padding: 5px 49px;
+                        margin: 16px 0px 0px;
+                        border: 1.5px solid #2e2e2e;
+                        background-color: #2e2e2e;
+                        color: #fff;
+                        font-size: 1.6rem;
+                        font-weight: bold;
+                    }
+                    .social {
+                        .column-left;
+                        // align-items: flex-start;
+                        ul {
+                            width: 100%;
+                            align-items: flex-start;
+                            li {
+                                padding: 10px 0px;
+                                a {
+                                    text-decoration: none;
+                                    font-size: 20px;
+                                    color: @clr-black;
+                                    .fab, .fa {
+                                        font-size: 24px;
+                                        padding-right: 15px;
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            /* Footer */
+            footer {
+                background-color: rgba(22, 22, 22, .5);
+                margin-top: 30px;
+                text-align: center;
+                padding: 25px 0px 40px;
+                color: #fff;
+                font-size: .8rem;
+            }
+        }
+    }
+/* Mobile Query */
+@media (max-width: 650px) {
+    html, body {
```

Comment:
While it's OK to structure your less this way, it is best practice to nest your media queries inside of your less structures, instead of recreating your less structure for every media query. This is called media query bubbling. Here is a link on it: http://alwaystwisted.com/articles/2012-05-05-everyday-im-bubbling-with-media-queries-and-less